articles/webcam-matlab/index.md

```diff
@@ -0,0 +1,148 @@
+
+```Matlab
+vidwriter = videowriter(‘myvideo.mp4’)
+open(vidwriter)
+```
+
+define a for loop from one to the number of frames. This defines the number of frames you want to capture
+```Matlab
+for index = 1: nof
+img = snapshot(cam); %this is to acquire the frame
+```
+
+we then write the video by executing the `writeVideo` function
+
+```Matlab
+writeVideo(vidWriter, img);
+pause(t/nof)
+end
+```
+
+by executing the `snapshot(cam)`, each snapshot will be stored in `img` and at the same time writing to vidWriter that we defined before. The `pause(t/nof)` command is a pause given proportionally to create the fps. 
+Once this is done, we close the vidWrite and clear the camera object.
+```Matlab
+close(vidWriter)
+clear cam
+```
+
+### Making mobile phone as an IP camera.
+To make your mobile phone an IP camera, you need a third-party android application by which you can convert your mobile phone as an IP camera.
+To find the android application, go to the google play store on your mobile phone and search for IPCAM. Note that you will find many applications. You only need to choose on and install but it is recommended to use IPCAM from Pavel since it is the most popular for converting mobile phones to IPCAM.
+
+![ipcam](/engineering-education/webcam-matlab/playstore.jpg)
+Once it is installed, follow the steps below to view the live stream video.
+- Scroll down and select the start server option.
+![starting the server](/engineering-education/webcam-matlab/server.jpg)
+- once the start server is selected, the live stream video is visible on your phone.
+Note that at the lower part of the screen, you can see the IPv4 IP addresses.
+![ip address](/engineering-education/webcam-matlab/ip_address.jpg)
+
+The IPv4 address will be used to access the camera in our Matlab and so it should be noted.
+This IP address can also be opened in your PC browser and see the live streaming from the mobile IP camera. This is done by simply copying your IP address by long-pressing your screen and pasting and searching in your browser. Note that the phone and the PC must be in the wifi network.
+
+
+### How to access the live video stream from your IP to your Matlab.
+We first create a camera object. When creating the camera object, we need the IP address of the IP camera.
+```Matlab
+>> cam3 = ipcam(‘http://192.168.1.4:8080/video);
+>> %execute the preview command to get the video on your Matlab
+>> preview(cam3)
+```
+
+### Conclusion
+Matlab is better software to use for live video streaming. This is because it has support packages that make this process easier. It also easier to use since the codes are not bulky and also easy to understand. IPCAM and WEBCAM are very important. This is because they apply to every sector to improve the security status. They can also be used to capture images for further analysis and data science. 
```

Comment:
If possible, add a `Further reading` section, which contains other reference links which might help the reader understand the concept better.